pkg/e2e/operators/cloudingress/certificate.go

```diff
@@ -0,0 +1,67 @@
+package cloudingress
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/constants"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// tests
+var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
+	h := helper.New()
+	var originalCert string
+	ginkgo.Context("publishingstrategy-certificate", func() {
+		ginkgo.It("IngressController should be patched when update Certificate", func() {
+			ingress1, _ := getingressController(h, "default")
+			originalCert = string(ingress1.Spec.DefaultCertificate.Name)
+			updateCertificate(h, "foo-bar")
+			time.Sleep(time.Duration(60) * time.Second)
+			ingress, _ := getingressController(h, "default")
+
+			temp := (int64(1))
+			Expect(string(ingress.Spec.DefaultCertificate.Name)).To(Equal("foo-bar"))
+			Expect(ingress.Generation == temp).To(Equal(false))
```

Comment:
```suggestion
			Expect(ingress.Generation == int64(1)).To(Equal(false))
```
Does this work? If so, I think this is a bit cleaner than declaring a single use variable.

Reply:
yep i'll fix that !